admin/css/archive-picker.css

```diff
@@ -115,22 +135,20 @@
   background-repeat: no-repeat;
 }
 
-.image-picker li img {
+.archive-picker-main li img {
   margin: 0;
   border: 2px solid transparent;
   box-sizing: border-box;
   width: 100%;
   object-fit: cover;
   cursor: pointer;
-  height: 220px;
 }
 
-.image-picker li {
-  display: inline-block;
+.archive-picker-main li {
```

Comment:
Looking at [this](https://github.com/greenpeace/planet4-master-theme/pull/2112/files#diff-61826b7e198488e0d8bd10ca420f37ebf03d0241dff3417b92560675f2142451R151), I think that only the `.archive-picker-main` should be positioned to `relative`. Not necessary the `li`.

Reply:
I think the `li` is `position: relative` so the _added-to-library_  `div` can be `position: absolute` within the `li`

Reply:
I split the whole component into 2

- _archive-picker-main_, houses the search bar and other form elements, the image list, help tooltip and the Spinner component.
- _archive-picker-sidebar_ houses the Sidebar component only.